internal/access/identity.go

```diff
@@ -80,6 +80,16 @@ func DeleteIdentity(c *gin.Context, id uid.ID) error {
 		return fmt.Errorf("delete identity access keys: %w", err)
 	}
 
+	groups, err := data.ListGroups(db, nil, data.ByGroupMember(id))
+	if err != nil {
+		return fmt.Errorf("delete identity from groups: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("deleted identity group membership: %w", err)
```

I'd like to have a different message here from the actual error removing the user, so we can differentiate them.